src/Layout/Page/Page.spec.tsx

```diff
@@ -0,0 +1,106 @@
+// REACT
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+// VENDOR
+import { shallow, mount } from 'enzyme';
+// COMPONENT
+import Page, {
+  Page as PageComponent,
+  DEFAULT_HEADER_HEIGHT,
+  DEFAULT_FOOTER_HEIGHT,
+  DEFAULT_LAYOUT_WIDTH,
+  TRANSPARENT,
+  DEFAULT_CONTENT_WIDTH,
+  // DEFAULT_CONTENT_WIDTH,
+  // DEFAULT_SIDEBAR_WIDTH,
+  // TRANSPARENT,
+} from './Page.component';
+
```

Comment:
`const {  DefaultLayout  } = Page;` can make things easier.

Reply:
Since I'm going to breaking up the components this shouldn't be an issue (soon).